### PR TITLE
Staging db

### PR DESCRIPTION
**Pull request by kvikshaug**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 *
 !src/
+!migrations/
```
Comment (ChristianLieven) [inline .dockerignore:3]: Oh, the migrations folder is not tracked?

**Review by ChristianLieven — APPROVED** [review]
Looks good to me! I hope it didn't take you long to figure this all out again!

**Comment by ChristianLieven** [comment]
Could the reason that travis is failing be this?:

https://stackoverflow.com/questions/43248455/gitlab-runner-local-build-login-from-non-tty-device

File: .dockerignore (hunk 1 above)
Comment (kvikshaug) [reply .dockerignore:3]: It's tracked in git, and this ensures it's also included in the Docker image.

**Comment by kvikshaug** [comment]
It's just that the docker hub credentials aren't added in the environment (error message is a bit weird). We'll set up Travis properly later!

**Pull request merged**